### ISSUE TITLE
class p-default is incorrectly added when using only :class special bind

### ISSUE THREAD

**Issue by liqueurdetoile** [hamed-ehtesham/pretty-checkbox-vue#35]
Hi,

When using only special class bind prop, the `mounted` hook doesn't detect that some classes have been provided and automatically apply `p-default` class. Therefore, some animations are disabled.

Here's a repro link : https://codepen.io/liqueurdetoile/pen/wvvZVaE

**Comment by marcosrocha85** [comment]
Isn't that a pretty checkbox issue itself? Not from pretty checkbox vue?

**Comment by liqueurdetoile** [comment]
@marcosrocha85 Nope. Issue is in mounted hook :

```javascript
if (this.$vnode.data && !this.$vnode.data.staticClass)
                this.default_mode = true;
```
It only check for static classes to fallback on `p-default` class. It should also check for binded classes. I think this one could do the trick :

```javascript
if (this.$vnode.data && !this.$vnode.data.staticClass && !this.$vnode.data.class)
                this.default_mode = true;
```
